server/parsec/components/memory/datamodel.py

```diff
@@ -855,4 +855,8 @@ class MemoryShamirShare:
 
 @dataclass(slots=True)
 class MemoryAccount:
-    user_email: str | None
+    user_email: EmailStr
+
+
+class UserAccountDataValidationModel(BaseModel):
```

Comment:
Used for data validation, I'm not sure that's the right way to do that

Reply:
You can add a test with a invalid email as `user_email`, I except `pydantic` to raise an exception.
But since they also provide `validate_email` we may have some surprise.

Anyway the best email validation is when we can send email to that address :shrug:

Reply:
Concerning the usage of that class, you want to use [`TypeAdapter`](https://docs.pydantic.dev/latest/concepts/type_adapter/) instead.